sandbox-template.md

```diff
@@ -0,0 +1,162 @@
+
+**_Becoming a sandbox project requires adoption of the CNCF IP Policy: [https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy)_**
+
+**_Note: there is a grace period after becoming a sandbox period to enable projects to adopt the policy, however, some prep is required to ensure there are no major blockers._**
+
+**_Has the IP policy been reviewed?_**
+
+**_List the repos for the project and their current license_**
+
+**_List any dependent repos (upstream/downstream) that are required to build the project  (including but not limited to  libraries, commercial tools, plugins)_**
+
+**_What actions are required to be compliant with the IP policy?_**
+
+
+# Other Considerations
+
+## **_Please note, these are not gating criteria but rather to:_**
+
+
+*   **_Collect a standard set of information for each project_**
+*   **_Provides a point in time capture of the state of the project which makes it easier to track progress at future reviews and / or promotion_**
+*   **_Help projects to prepare for SIG and TOC presentation_**
+*   **_Allow the SIG to review the project and perform due diligence for incubation_**
+*   **_Provide the TOC with the information required to accept sponsorship of a project and/or votes_**
+*   **_Identify and rectify any significant issues / blockers prior to presenting to the TOC and acceptance as a CNCF project_**
+
+
+## Cloud Native
+
+**_Does the project meet the definition of Cloud Native?  The CNCF charter states:_**
+
+
+    “Cloud native technologies empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach.
+
+
+    “These techniques enable loosely coupled systems that are resilient, manageable, and observable. Combined with robust automation, they allow engineers to make high-impact changes frequently and predictably with minimal toil.”
+
+
+## Project and Code Quality
+
+**_Are there any metrics around code quality?  Are there good examples of code reviews? Are there enforced coding standards?_**
+
+**_What are the performance goals and results? What performance tradeoffs have been made?  What is the resource cost?_**
+
+**_What is the CI/CD system?  Are there code coverage metrics?  What types of tests exist?_**
+
+**_Is there documentation?_**
+
+**_How is it deployed?_**
+
+**_How is it orchestrated?_**
+
+**_How will the project benefit from acceptance into the CNCF?_**
+
+**_Has a security assessment by the security SIG been done? If not, what is the status/progress of the assessment?_**
+
+
+# Promotion to Incubation 
+
+## Open Governance
+
+**_How are committers chosen?_**
+
+**_How are architectural and roadmap decisions made?_**
+
+**_How many decision makers are outside the sponsoring organization._**
+
+## Adoption
```

Comment:
This section seems like it should be in the proposal -- the project would be answering this, rather than SIG